src/components/App.tsx

```diff
@@ -1028,6 +1030,9 @@ class App extends React.Component<ExcalidrawProps, AppState> {
   });
 
   private onCopy = withBatchedUpdates((event: ClipboardEvent) => {
+    if (this.lastMouseDownTarget !== this.canvas) {
+      return;
+    }
```

Comment:
It would be best if we can come up with a different heuristic. You can focus/blur the excalidraw compo in ways other than `mousedown` (if nothing else, we should use `pointerdown` event, not use mouse-specific events).

Reply:
I spent some time digging into this and couldn't find a better way to check if the canvas is focused. 
However, I think we can go ahead with the approach and update the approach once we have a better way to handle this.

Reply:
I really think we should be conservative here and try to avoid and false negatives, and instead bias towards making sure Excalidraw editor works correctly first, and only then try to fix specific cases of not being able to copy outside.

Thus, allowing to copy text by checking `document.getSelection()` as I suggested below seems like a good start.

Reply:
sounds good 👍